### PR TITLE
Add additional Enterprise DB details

### PR DESCRIPTION
**Pull request by nickanderson**
Improve some details:
- lastseeninterval is measured in minutes.
- LastSeenHosts table is derived from the lastseen database.
- Host table IP address is derived from lastseen database for incoming connection

### CODE REVIEW

File: reference/enterprise-api-ref/sql-schema.markdown [hunk 1]
```diff
@@ -500,14 +508,15 @@ Information about communication between CFEngine clients.
...
+
 * **RemoteHostIP** *(text)*
     IP address of the remote host.
 
 * **LastSeenTimeStamp** *(timestamp)*
     Time when the connection was established.
 
 * **LastSeenInterval** *(real)*
-    Frequency within which both hosts open connection.
+    Average frequency in minutes between connections (both INCOMING and OUTGOING) with the host.
```
Comment (maciejmrowiec) [inline reference/enterprise-api-ref/sql-schema.markdown]: After fix in master this is in expressed in **seconds**, incoming and outgoing end up in separate entries (LastSeenDirection column), interval is for specific direction.

Comment (nickanderson) [inline reference/enterprise-api-ref/sql-schema.markdown]: OK fixed and squashed. Please approve new wording.

File: reference/enterprise-api-ref/sql-schema.markdown [hunk 2]
```diff
@@ -486,7 +492,9 @@ changetimestamp | 2015-03-12 14:52:36+00
 
 ### Table: LastSeenHosts
 
-Information about communication between CFEngine clients.
+Information about communication between CFEngine clients. Effectively a snapshot
+of each hosts lastseen database (`cf_lastseen.lmdb`, `cf-key -s`) at the time of
+their last report.
```
Comment (maciejmrowiec) [inline reference/enterprise-api-ref/sql-schema.markdown]: their last reported cf-agent execution

Comment (maciejmrowiec) [inline reference/enterprise-api-ref/sql-schema.markdown]: otherwise looks much better

**Pull request merged**